### PR TITLE
Oas30 import fixes

### PR DESCRIPTION
**Pull request by kokyhin**

### CODE REVIEW

File: src/oas/oas-to-ms3/oas-30-to-ms3/index.ts [hunk 1]
```diff
@@ -55,23 +55,42 @@ class MS3toOAS30toMS3 {
...
     if (info.description) settings.description = info.description;
 
     return settings;
   }
 
   convertOperations(operations: OAS30Interface.PathItemObject | any): MS3Interface.Method[] {
     const methodsKeys = ['get', 'post', 'put', 'delete', 'options', 'head', 'patch'];
-
-    return methodsKeys.reduce((methodsArray: any[], methodKey: string) => {
+    const foundTraitsName = <any>[];
```
Comment (eliasmamo) [inline src/oas/oas-to-ms3/oas-30-to-ms3/index.ts]: why not foundTraitsName: string[] = [];

File: src/oas/oas-to-ms3/oas-30-to-ms3/index.ts [hunk 2]
```diff
@@ -55,23 +55,42 @@ class MS3toOAS30toMS3 {
...
+    const methods = methodsKeys.reduce((methodsArray: any[], methodKey: string) => {
```
Comment (eliasmamo) [inline src/oas/oas-to-ms3/oas-30-to-ms3/index.ts]: const methods = methodsKeys.reduce((methodsArray: MS3Interface.Method[], methodKey: string) => {

File: src/oas/oas-to-ms3/oas-30-to-ms3/index.ts [hunk 3]
```diff
@@ -55,23 +55,42 @@ class MS3toOAS30toMS3 {
...
 
       const method: MS3Interface.Method = this.convertOperation(operation, methodKey);
       methodsArray.push(method);
+      foundTraitsName.push(method.name.toLowerCase());
 
       return methodsArray;
     }, []);
+    const missedMethods = difference(methodsKeys, foundTraitsName);
+    each(missedMethods, (methodName) => {
+      const template = {
```
Comment (eliasmamo) [inline src/oas/oas-to-ms3/oas-30-to-ms3/index.ts]: const template: MS3Interface.Method = {
and remove casting inside

File: src/oas/oas-to-ms3/oas-30-to-ms3/index.ts (hunk 1 above)
Comment (kokyhin) [reply src/oas/oas-to-ms3/oas-30-to-ms3/index.ts]: [ts] 'string' only refers to a type, but is being used as a value here.

**Pull request merged**